integration_test/cases/browser/shadow_dom_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Wallaby.Integration.Browser.ShadowDomTest do
```

Comment:
I believe you need to add this test to `integration_tests/tests.exs` for them to actually run.

Reply:
Thanks!